src/pages/library/DataBrowser.js

```diff
@@ -173,6 +173,7 @@ const makeDataBrowserTableComponent = ({ sort, setSort, selectedData, toggleSele
       useHover: false,
       underRowKey: 'underRow',
       rows: _.map(datum => {
+        datum['access'] = 'Controlled'
```

Comment:
Please remove this line! Phil has a code change to set some of the datasets to Controlled